### DISCUSSION TITLE
how can i use created, mounted in setup script

### DISCUSSION THREAD

**Discussion by shahensaifullah** [nuxt/nuxt#17331] [General]
in setup script, there is any way to write created and mounted in setup script method?

**Comment by viii47** [comment]
You can use the new [lifecycle hooks of vue3](https://vuejs.org/api/composition-api-lifecycle.html#onmounted) to stipulate those functions. 

 ### example
```js
// index.vue
<script setup lang="ts">
onMounted(() => {
    // your code
})
</script>
```
